public/locales/en/components.json

```
@@ -129,6 +129,12 @@
     "property-label": {
       "read-only-tooltip": "This property is read-only, meaning that it's not intended to be changed.",
       "read-only-label": "Read-only"
+    },
+    "confirmation-modal": {
+      "title": "Confirm Change",
+      "description": "You are about to change the value of '{{propertyName}}'. This property is marked as sensitive and requires confirmation. Do you want to proceed?",
```

Is "sensitive" the right word here?  Can't come up with a better one, though. "protected"? But maybe that's also confusing...

It's difficult to come up with the description here since we can't generalize based on what type of property we have applied this setting to... Also, depending on the setting, this modal may always show, even for properties that are not marked  as sensititve/protected/dangerous to change.  So, how about just removing this part? 

```suggestion
      "description": "You are about to change the value of '{{propertyName}}'. Do you want to proceed?",
```
or 
```suggestion
      "description": "You are about to change the value of '{{propertyName}}', which requires confirmation. Do you want to proceed?",
```

---

I think `This property requires confirmation.` is probably unambiguous enough?

---

Yep wasn't too sure on this text, going with `This property requires confirmation.`